statusHandler/appStatusFacade.go

```diff
@@ -0,0 +1,58 @@
+package statusHandler
+
+import "github.com/ElrondNetwork/elrond-go/core"
+
+// AppStatusFacade will be used for handling multiple monitoring tools at once
+type AppStatusFacade struct {
+	handlers []core.AppStatusHandler
+}
+
+// NewAppStatusFacadeWithHandlers will receive the handlers which should receive monitored data
+func NewAppStatusFacadeWithHandlers(aphs ...core.AppStatusHandler) AppStatusFacade {
+	var appStatusFacade AppStatusFacade
+	appStatusFacade.handlers = aphs
```

Comment:
I think you should check here for nils (aphs[i] should not be nil)

Reply:
done

Reply:
Not done here